src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -1287,18 +1285,26 @@ Containers::Optional<TextureData> TinyGltfImporter::doTexture(const UnsignedInt
             minFilter = SamplerFilter::Linear;
             mipmap = SamplerMipmap::Linear;
             break;
-        default: CORRADE_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
+        default:
```

Comment:
Maybe add `case -1` right below `case TINYGLTF_TEXTURE_FILTER_LINEAR_MIPMAP_LINEAR` and keep an unreachable default instead? So we stay as strict as before.

Same below, I think this could also fix the minor coverage loss.